cmd/bake/bake.go

```diff
@@ -67,19 +70,74 @@ type bakePostRequest struct {
 	Wait bool   `json:"wait,omitempty"`
 }
 
+type repos struct {
+	URLs []string `yaml:"repos"`
+}
+
+type bakeStatus struct {
+	URL string
+	Err error
+}
+
 func run(opts *Options) error {
-	bodyPostReq := &bakePostRequest{
-		URL:  opts.URL,
-		Wait: opts.Wait,
+	var repos repos
+	uniqueURLs := make(map[string]int)
+
+	if opts.FilePath != "" {
+		configFile, err := os.ReadFile(opts.FilePath)
+
```

Comment:
nit: Unnecessary newline spacing here. Typically error blocks appear right after where the error was generated in Go.